*** Starting uWSGI 2.0.15 (64bit) on [Thu May 10 11:46:36 2018] ***
compiled with version: 5.4.0 20160609 on 11 November 2017 10:45:41
os: Linux-4.13.0-39-generic #44~16.04.1-Ubuntu SMP Thu Apr 5 16:43:10 UTC 2018
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/atguigu/work/information_push
writing pidfile to uwsgi.pid
detected binary path: /home/atguigu/.virtualenvs/py2/bin/uwsgi
chdir() to /home/atguigu/work/information_push/
your processes number limit is 7677
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:9001 fd 3
Python version: 2.7.12 (default, Nov 20 2017, 18:23:56)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x7d29a0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 415280 bytes (405 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
unable to load app 0 (mountpoint='') (callable not found or import error)
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x7d29a0 pid: 66351 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 66351)
spawned uWSGI worker 1 (pid: 66354, cores: 2)
spawned uWSGI worker 2 (pid: 66355, cores: 2)
spawned uWSGI worker 3 (pid: 66356, cores: 2)
spawned uWSGI worker 4 (pid: 66357, cores: 2)
*** Starting uWSGI 2.0.15 (64bit) on [Thu May 10 11:47:15 2018] ***
compiled with version: 5.4.0 20160609 on 11 November 2017 10:45:41
os: Linux-4.13.0-39-generic #44~16.04.1-Ubuntu SMP Thu Apr 5 16:43:10 UTC 2018
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/atguigu/work/information_push
writing pidfile to uwsgi.pid
detected binary path: /home/atguigu/.virtualenvs/py2/bin/uwsgi
chdir() to /home/atguigu/work/information_push/
your processes number limit is 7677
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:9001).
bind(): Address already in use [core/socket.c line 769]
